src/main/scala/encry/http/HttpServer.scala

```diff
@@ -0,0 +1,13 @@
+package encry.http
+
+import akka.http.scaladsl.Http
+import encry.MVP2._
+import encry.Utils.Settings
+
+object HttpServer {
+
+  def start(): Unit = {
+    val (host, port): (String, Int) = (Settings.settings.httpHost, Settings.settings.httpPort)
+    Http().bindAndHandle(ApiRoute().apiInfo, host, port)
```

Comment:
You can write:
 Http().bindAndHandle(ApiRoute().apiInfo, Settings.settings.httpHost, Settings.settings.httpPort)